### PR TITLE
week1/Assignment2: Implemented html code of catty board

### PR DESCRIPTION
**Pull request by developervinit**
Exercise1.2:(2-3hours)- Use "Lato" font for entire text in UI- Use line awesome Icon library for all icons- Cattyboard album image URL:https://vif1g.csb.app/src/assets/cattyboard.jpg- Icons used in assignment:la-barsla-homela-searchla-volume-upla-userla-cogla-spotifyla-soundcloud
la-playla-plusla-ellipsis-hla-step-backwardla-step-forwardla-undo-altla-random- Note: Feel free to use any other font library if you wishGuidelines:1.There should be a separate branch created in GitHub for this exercise.2.The UI should closely match with the screengrab given as sample. This includes thecolors, fonts, icons, text etc.3.Semantic tags should be used for laying out the UI.4.There should be one or more Git commits with meaningful commit messages.

### CODE REVIEW

File: Week-1/assignment2/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Catty Boeard</title>
```
Comment (sairamin) [inline Week-1/assignment2/index.html:9]: Spelling mistake

File: Week-1/assignment2/index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+</head>
+<body>
+    <div class="super_Container">
```
Comment (sairamin) [inline Week-1/assignment2/index.html:12]: superContainer
SuperContainer
super_container

Please use any of the above casing

File: Week-1/assignment2/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                <li class="la la-home"></li>
+                <li class="la la-search"></li>
+                <li class="la la-volume-up"></li>
+                <li class="la la-user"></li>
+                <li class="la la-cog"></li>
+                <li class="la la-spotify"></li>
+                <li class="la la-soundcloud"></li>
+            </ul>
+        </nav>
+        <div class="cntnt_Container">
```
Comment (sairamin) [inline Week-1/assignment2/index.html:25]: content

File: Week-1/assignment2/index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                <li class="la la-step-backward"></li>
+                <li class="la la-stop"></li>
+                <li class="la la-step-forward"></li>
+                <li class="la la-undo-alt"></li>
+                <li class="la la-random"></li>
+            </ul>
+        </div>
+    </footer>
+</body>
+</html>
```
Comment (sairamin) [inline Week-1/assignment2/index.html:90]: Add extra line at the end

File: Week-1/assignment2/style.css [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+.nav_Container > ul{
+    display: contents;
+}
+
+.nav_Container > ul > li{
+    font-size: 20px;
+    text-align: center;
+    margin: 42px 0px 42px 0px;
```
Comment (sairamin) [inline Week-1/assignment2/style.css:26]: margin: 42px 0px

File: Week-1/assignment2/style.css [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+        padding: 0px;
+        text-align: center;
+    }
+    .ftr_Right > ul > li{
+        margin: 10px;
+    }
+    .ftr_Left > :nth-child(2){
+        margin-left: 0px;
+    }
+  }
```
Comment (sairamin) [inline Week-1/assignment2/style.css:141]: new line

File: Week-1/assignment2/index.html (hunk 1 above)
Comment (mrkarthik3) [reply Week-1/assignment2/index.html:9]: @developervinit  Please create a screen shot of the project's final look... add that to project folder.... and push that to GitHub.

Difficult to judge if this code matches with the required look.

File: Week-1/assignment2/index.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+                    <div class="hdr_Media_Wraper">
+                        <span class="las la-play">Play All</span>
+                        <span class="las la-plus">Add to</span>
+                        <span class="las la-ellipsis-h">More</span>
+                    </div>
+                </div>
+            </header>
+            <section class="content">
+                <table class="table">
+                    <tr class="tr">
```
Comment (mrkarthik3) [inline Week-1/assignment2/index.html:42]: Please add applicable semantic tags for table... like thead/tbody etc...

File: Week-1/assignment2/style.css [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+.super_Container{
+    display: flex;
+    width: 100%;
+}
+
+/* for nav bar */
+.nav_Container{
+    background-color: #f0f0f0;
+    display: flex;
+    padding: 10px;
```
Comment (mrkarthik3) [inline Week-1/assignment2/style.css:15]: Was instructed by our mentor to use multiples of 4 for properties like padding/margin/font-size etc....
That is considered professional.

File: Week-1/assignment2/style.css [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+
+/* for header  */
+.hdr{
+    background-color: #454545;
+    padding: 32px;
+    display: flex;
+    color: white;
+}
+.hdr_Cntnt_Wraper{
+    margin-left: 35px;
```
Comment (mrkarthik3) [inline Week-1/assignment2/style.css:43]: Please apply "multiples of 4" principle here... Make it 36 or 32 px

File: Week-1/assignment2/style.css [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+}
+.hdr_Cntnt_Wraper > h1{
+    margin-top: 0px;
+}
+.hdr_Cntnt_Wraper > p{
+    margin-top: 4px;
+    margin-bottom: 0px;
+}
+.hdr_Media_Wraper{
+    margin-top: 126px;
```
Comment (mrkarthik3) [inline Week-1/assignment2/style.css:53]: Please apply "multiples of 4" principle here...

File: Week-1/assignment2/style.css [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+    padding: 4px;
+    cursor: pointer; 
+}
+
+/* for footer */
+.footer{
+    width: 100%;
+    background-color: #43689b;
+    display: flex;
+    flex-direction: row;
```
Comment (mrkarthik3) [inline Week-1/assignment2/style.css:84]: "flex-direction: row" is redundant.... Isn't it ?
Why mention it... It is "row" by default anyway

File: Week-1/assignment2/style.css [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+.ftr_Right{
+    width: 60%;
+}
+.ftr_Right > ul{
+    list-style: none;
+    float: right;
+}
+.ftr_Right > ul > li{
+    display: inline;
+    margin: 0px 10px 0px 140px;
```
Comment (mrkarthik3) [inline Week-1/assignment2/style.css:108]: Please apply "multiples of 4" principle here...

**Review by mrkarthik3 — CHANGES_REQUESTED** [review]
Please add a screenshot of the project's final look.

**Pull request merged**